builder/builder.py

```diff
@@ -0,0 +1,138 @@
+import click
+import os
+import sys
+import logging
+
+TIMESTAMP_FORMAT = '%Y-%m-%dT%H:%M:%S'
+SHORT_TIMESTAMP_FORMAT = '%Y-%m-%d'
+
+logger:object = logging.getLogger('Artifact Builder ===>')
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+class builder:
+    def __init__(self, *args, **kwargs) -> None:
+        print('CMR Query object created')
```

Comment:
Should probably update print statement to remove CMR reference.

Reply:
work for building github public project should be PODAAC-2551 instead of PODAAC-2541.  Closing this PR soon and reopening another one.